src/layouts/PersonDetails/index.js

```diff
@@ -7,41 +7,84 @@ import {
   SubdivTitle,
   TileContent,
 } from "./styled";
-import poster from "../../common/images/poster.png";
-import person from "../../common/images/person.png";
 import { Tile } from "../Tile/index";
+import { useSelector } from "react-redux";
+import {
+  selectPeopleCredits
+} from "../../features/peoples/peopleDetails/peopleDetailsSlice";
+import { selectPosterSizes, selectProfileSizes } from "../../features/configurationSlice";
 export const PersonDetails = () => {
+  const peopleCredits = useSelector(selectPeopleCredits);
+  const profileSizes = useSelector(selectProfileSizes);
+  const posterSizes = useSelector(selectPosterSizes);
+
   return (
     <>
       <ContentBackground>
         <Content>
           <TileContent>
-            <Tile personDetails={"personDetails"} srcImage={person} />
+            <Tile personDetails profileSizes={profileSizes} />
           </TileContent>
           <CastContent>
-            <SubdivTitle>Movies - cast()</SubdivTitle>
+            <SubdivTitle>
+              Movies - cast({peopleCredits.cast && peopleCredits.cast.length})
```

Comment:
People credits going out of tile

Reply:
What do you have on mind?

Reply:
I indicate in which component the description goes beyond the tile.

Reply:
You have on mind mobile view?

Reply:
that's right, it was mobile view

Reply:
OK, I will fix it tommorow.